http-server/server/server.go

```diff
@@ -17,21 +18,59 @@ import (
 type serverInfo struct {
 	pageAccessCount int
 	param           string
+	variables       map[string]string
+}
+
+// LoadSettings reads data from a file called "env" in current dir
+// and sets all environment variables for this run.
+// Returns map of keys&values in the env file.
+func LoadSettings() (vars map[string]string, err error) {
```

Comment:
You could move this to the constructor, ie `NewServerInfo()` and make this function private :)

Reply:
good point! reworked now